packages/electrode-archetype-react-app/lib/logger.js

```diff
@@ -0,0 +1,61 @@
+const winston = require("winston");
+const bunyan = require("bunyan");
+
+const archetype = require("electrode-archetype-react-app/config/archetype");
+const enableWinstonLog = archetype.webpack.enableWinstonLog;
+
+const winstonLogger = new (winston.Logger)({
+  exceptionHandlers: [
+    new (winston.transports.File)({
+      filename: "exceptions.log"
+    })
+  ],
+  transports: [
+    new (winston.transports.Console)({
+      colorize: true,
+      level: 'info',
+      prettyPrint: true
+    }),
+    new (winston.transports.File)({
+      name: 'error-file',
+      filename: 'winston-error.log',
+      level: 'error'
+    }),
+    new (winston.transports.File)({
+      name: 'warn-file',
+      filename: 'winston-warn.log',
+      level: 'warn'
+    }),
+    new (winston.transports.File)({
+      name: 'info-file',
+      filename: 'winston-info.log',
+      level: 'info'
+    }),
+    new (winston.transports.File)({
+      name: 'verbose-file',
+      filename: 'winston-verbose.log',
+      level: 'verbose'
+    })
+  ]
+});
+
+const bunyanLogger = bunyan.createLogger({
+  name: "electrode",
+  src: true,
+  streams: [
+    {
+      level: 'error',
+      path: "bunyan-error.log"
+    },
+    {
+      level: 'warn',
+      path: "bunyan-warn.log"
+    },
+    {
+      level: 'info',
+      path: "bunyan-info.log"
+    }
+  ]
+});
+
+module.exports = enableWinstonLog ? winstonLogger : bunyanLogger;
```

Comment:
for this scenario, our purpose is to easily show nicely formatted messages that allow users to identify useful information quickly.  Our use is mainly display to terminal and file.  Other advance features such as remote transport etc are not important to use.  So the logger that meets that usage pattern is the preferred one.